aws_clean.py

```diff
@@ -376,6 +376,65 @@ def delete_hosted_zones(service_name):
             client.delete_hosted_zone(Id=hz['Id'])
 
 
+def delete_iam_roles(service_name):
+    logging.info('Deleting IAM roles')
+    client = boto3.client('iam')
+    response = client.list_roles(MaxItems=1000)
```

Comment:
Is the assumption here that there should never be more than `1000` roles at any given time? Reason I ask is because it looks like if there are say, `1001` roles, only `1000` will be nuked. In saying that, since you manually cleaned these up recently I'd like to think that now this script is in place we should never be in a situation again where we have > `1000` to clean up.

Reply:
You're right. Let me introduce some pagination instead.